electron-builder-appx.json

```diff
@@ -29,7 +29,5 @@
       "name": "simplenote",
       "schemes": ["simplenote"]
     }
-  ],
-  "afterSign": "./after_sign_hook.js",
-  "afterAllArtifactBuild": "./after_sign_hook.js"
```

Comment:
Notice that these were the same script. If you look into it, the only thing it does is notarizing for macOS. I decided to remove it to keep the config as small as possible.

Reply:
To check my understanding of this file: it's fine to remove the calls to the `after_sign_hook.js` script because they only do macOS notarization and this file only deals with the Windows AppX build, correct?

Reply:
@spencertransier correct